basic-auth/README.md

```diff
@@ -0,0 +1,68 @@
+# Basic Authentication
+
+
+This sample allows you to authenticate an incoming request with a Basic Authentication header using a `client_id` and a `client_secret` as the encoded credential pair. 
+
+## Prerequisites
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro)
+2. Configure [external access](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing#external-access) for API traffic to your Apigee X instance
+3. Access to deploy proxies, create products, apps and developers in Apigee
+4. Make sure the following tools are available in your terminal's $PATH (Cloud Shell has these preconfigured)
+   * [gcloud SDK](https://cloud.google.com/sdk/docs/install)
+   * unzip
+   * curl
+   * jq
+   * npm
+
+# (QuickStart) Setup using CloudShell
+
+Use the following GCP CloudShell tutorial, and follow the instructions.
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://ssh.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/ra2085/apigee-usecase-mashup&cloudshell_git_branch=main&cloudshell_workspace=.&cloudshell_tutorial=basic-auth/docs/cloudshell-tutorial.md)
```

Comment:
should point to the GoogleCloudPlatform/apigee-samples repo

Reply:
fixed. was testing.